later/lib/presentation/later_app.dart

```diff
@@ -13,9 +16,10 @@ class LaterApp extends StatelessWidget {
       title: 'later',
       routerDelegate: _router.delegate(),
       routeInformationParser: _router.defaultRouteParser(),
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
+      theme: ThemeData().copyWith(extensions: <ThemeExtension<dynamic>>[
```

Comment:
It is not urgent to specify the type, the compiler will figure it out 

```suggestion
      theme: ThemeData().copyWith(extensions: [
```

Reply:
@hardioi ☝️